test/e2e/util.go

```diff
@@ -491,8 +491,8 @@ func createTestingNS(baseName string, c *client.Client) (*api.Namespace, error)
 }
 
 // deleteTestingNS checks whether all e2e based existing namespaces are in the Terminating state
-// and waits until they are finally deleted.
-func deleteTestingNS(c *client.Client) error {
+// and waits until they are finally deleted. It ignores namespace skip.
+func deleteTestingNS(c *client.Client, skip string) error {
```

Comment:
I really really really don't like this function, for the following reasons:

1. It's called deleteTestingNS, but it doesn't actually delete anything.  All it does is do some checks.
2. It takes a namespace as a parameter, and given the name of the function, callers could reasonably expect that to be the namespace upon which it operates, but it's in fact the opposite of that.

Could you not rename it to something like checkTestingNSDeletedExcept().  I know it's slightly verbose, but at least it's true :-)